promptsource/templates/paws-x/en/templates.yaml

```diff
@@ -1,203 +1,205 @@
 dataset: paws-x
 subset: en
 templates:
-  289448c3-63fc-4904-8387-86d232424243: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 289448c3-63fc-4904-8387-86d232424243
-    jinja: "{% if label == 1 %} \nParaphrase the sentence: {{sentence2}} \n||| \n\
-      {{sentence1}} \n{% endif %}"
-    metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: false
-      choices_in_prompt: null
-      metrics: []
-      original_task: false
-    name: paraphrase-task-reverse
-    reference: Create a generative paraphrase task
-  2ff509a8-2712-4406-97bd-ce86824cdc18: !Template
-    answer_choices: null
+  0be7cecd-b427-4ec9-9b0e-666d6dae00dd: !Template
+    answer_choices:
+    - 'No'
```

Comment:
@stephenbach changed the answer choices format so you can write this as  "No ||| Yes" now. Stephen are we keeping the old format as well?

Reply:
i thought we were merging all `answer_choices` and `answer_choices_key` for all templates later